device/arista/x86_64-arista_7050_qx32s/Arista-7050-QX-32S/qos.json

```diff
@@ -115,24 +115,23 @@
         }
     },
     "WRED_PROFILE": {
-        "AZURE_LOSSY" : {
+        "AZURE_LOSSLESS" : {
             "wred_green_enable":"true",
             "wred_yellow_enable":"true",
+            "wred_red_enable":"true",
             "ecn":"ecn_all",
-            "red_max_threshold":"512000",
-            "red_min_threshold":"512000",
-            "yellow_max_threshold":"512000",
-            "yellow_min_threshold":"512000",
-            "green_max_threshold": "184320",
-            "green_min_threshold": "184320"
+            "red_max_threshold":"312000",
+            "red_min_threshold":"104000",
```

Comment:
are we sure we want to different value for min and max?

Reply:
This is based on the test report recommendation from other team.